src/compiler/core.ts

```diff
@@ -1659,7 +1659,7 @@ namespace ts {
     }
 
     export function isRootedDiskPath(path: string) {
-        return getRootLength(path) !== 0;
+        return path && getRootLength(path) !== 0;

```

Comment:
so why do we have undefined file paths?

Reply:
Answers from plugins may include them

Reply:
would not that cause issues in other places as well, and should we just prune/sanitize the list first in these cases?